ipynb/primary-processing/11-generate_per_bodysite_results.md

```diff
@@ -0,0 +1,253 @@
+Analysis focused on a paricular dataset often starts with seperating the data by bodysite. This notebook, and subsequent notebooks (12-15) will seperate the American Gut dataset by bodysite and rarefaction depth.
+
+```python
+import itertools
+import os
+
+import numpy as np
+import pandas as pd
+
+import americangut.util as agu
+import americangut.notebook_environment as agenv
+
+chp_path = agenv.activate('11-packaged')
+```
+
+We're also going to set up a series of reference variables, which we'll use through this analysis.
+
+```python
+alpha_metrics = {'pd': 'PD_whole_tree',
+                 'chao1': 'chao1',
+                 'shannon': 'shannon',
+                 'observedotus': 'observed_otus'
+                }
+depths = agenv.get_rarefaction_depth()
+depth_names = ['1k', '10k']
+```
+
+We'll start by splitting the raw OTU table and metadata by the `BODY_HABITAT` field, so we have an OTU table for each bodysite.
+
+```python
+for trim in ['notrim', '100nt']:
+    for depth in ['raw', '1k', '10k']:
+        meta = agu.get_existing_path(agenv.paths['package']['split']['%s-map' % trim])
+        otus = agu.get_existing_path(agenv.paths['package']['split']['%s-%s-otu' % (trim, depth)])
+        odir = agu.get_new_path(agenv.paths['package']['split']['%s-%s-dir' % (trim, depth)])
+
+        !mkdir -p $odir
+        !split_otu_table.py -i $otus \
```

Comment:
But, it doesn't have the accompanying filtered metadata. There's an advantage in my mind to having it split the metadata, too. But, I can also go back to the tables from collapsed.